gazebo_ros/package.xml

```diff
@@ -32,7 +32,12 @@
   <depend>std_srvs</depend>
   <depend>tinyxml_vendor</depend>
 
+  <exec_depend>ament_index_python</exec_depend>
+  <exec_depend>geometry_msgs</exec_depend>
   <exec_depend>launch_ros</exec_depend>
+  <exec_depend>python3-catkin-pkg</exec_depend>
```

Comment:
are we sure we want `python3-catkin-pkg` in the `foxy` branch?

Reply:
It is imported here:

https://github.com/ros-simulation/gazebo_ros_pkgs/blob/3f18465d3fac30c729f9406eff0fda67ab606502/gazebo_ros/scripts/gazebo_ros_paths.py#L32

Reply:
I suppose this is fine, though I think we could replace this logic with equivalent ament API calls in a follow-up.

Reply:
oh, I guess it's even used in ros2pkg (https://github.com/ros2/ros2cli/pull/94):

* https://github.com/ros2/ros2cli/blob/5c99e5f777ea3355ad8760663e3ee486872e2600/ros2pkg/ros2pkg/verb/create.py#L23-L26